### PR TITLE
x11-themes/papirus-icon-theme: skip pantheon variants by default

### PR DESCRIPTION
**Pull request by ixti**
ePapirus and ePapirus-Dark are variants designed for elementaryOS and Pantheon DE.

Signed-off-by: Alexey Zapparov <alexey@zapparov.com>

### CODE REVIEW

File: x11-themes/papirus-icon-theme/papirus-icon-theme-20211001-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86"
+IUSE="pantheon"
+
+src_compile() { :; }
+
+src_install() {
+	insinto /usr/share/icons
+	doins -r Papirus{,-Dark,-Light}
+	use pantheon && doins -r ePapirus{,-Dark}
```
Comment (juippis) [inline x11-themes/papirus-icon-theme/papirus-icon-theme-20211001-r1.ebuild:22]: In general we don't add USE flags to control installing few kb worth of stuff, 
https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0301

but even without that, why would you not want to install these variants unconditionally? :P

Comment (ixti) [reply x11-themes/papirus-icon-theme/papirus-icon-theme-20211001-r1.ebuild:22]: I'm fine to install those unconditionally. I simply was always confused what exactly ePapirus theme is (it's not very obvious from its name), and according to README that flavour is sensible for the pantheon desktop environment only.

@juippis do you think we should remove use flag?

Comment (ixti) [reply x11-themes/papirus-icon-theme/papirus-icon-theme-20211001-r1.ebuild:22]: I've opened a new PR with new version of the package and unconditional theme installation (but added a comment in the ebuild what those themes are) :D

Comment (ixti) [reply x11-themes/papirus-icon-theme/papirus-icon-theme-20211001-r1.ebuild:22]: But, I think we need the use flag :P

**Comment by ixti** [comment]
Closing in favour of https://github.com/gentoo/gentoo/pull/22893

**Pull request closed without merging**